gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/resources/users/UserEndpoint.java

```diff
@@ -114,7 +120,7 @@ public void update(RoutingContext context) {
 
             // schemas field is REQUIRED and MUST contain valid values and MUST not contain duplicate values
             try {
-                checkSchemas(user.getSchemas(), EnterpriseUser.SCHEMAS);
+                checkSchemas(user.getSchemas(), schemas);
```

Comment:
Here we are performing schema validation using the schemas extracted from the payload. 
I wondering if we should create a constant of Supported Schemas (with User, Entreprise and Custom) and use it here :thinking:

Reply:
FP